x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/method_steps.tsx

```diff
@@ -56,16 +55,8 @@ export const BUILD_SEARCH_EXPERIENCE_STEP: EuiContainedStepProps = {
       <p>
         <FormattedMessage
           id="xpack.enterpriseSearch.content.newIndex.steps.buildSearchExperience.content"
-          defaultMessage="After building your connector, your content is ready. Build your first search experience with {elasticsearchLink}, or explore the search experience tools provided by {appSearchLink}. We recommend that you create a {searchEngineLink} for the best balance of flexible power and turnkey simplicity."
+          defaultMessage="After building your connector, your content is ready to create a {searchApplicationLink} with {elasticsearchLink}."
```

Comment:
@julianrosado FYI I think this page will not exist when new [redesign PR ](https://github.com/elastic/kibana/pull/155149) would be merged. Added it for a safer side.